oak_functions/proto/response.proto

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright 2021 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package oak.functions.response;
+
+// Represents the response from an Oak Functions application. This is included in the body of the
+// HTTP response sent to the client.
+message FunctionsResponse {
```

Comment:
These objects have names that repeat parts of the package name. Also I think we would want to also have a corresponding Request message at some point (e.g. to combine body and policy). So how about fully qualified name of `oak.functions.invocation.{Response,StatusCode}` ?

Reply:
I am not sure about `invocation`. To me, it sounds like a very oak-specific concept based on the concept of channels. I've removed `Functions` from the names.

Reply:
Oh, I just meant it as "function invocation" anyways, that's why the endpoint is called `/invoke` already. This is fine anyways.